src/Claim/Claims.php

```diff
@@ -58,12 +59,12 @@ private function getGuidKey( $guid ) {
 	}
 
 	/**
-	 * @param Claim $claim
+	 * @param Statement $claim
```

Comment:
You do realize you've -1'd stuff several times because it changed the type and not the var name?